jupyterhub/spawners.py

```diff
@@ -318,10 +318,10 @@ def get_pod_manifest(self):
         }
         # some annotations are repeated as labels so that the k8s api can filter resources
         self.extra_labels = {
-            RENKU_ANNOTATION_PREFIX + "username": safe_username,
+            RENKU_ANNOTATION_PREFIX + "username": safe_username[:63],
             RENKU_ANNOTATION_PREFIX + "commit-sha": options.get("commit_sha"),
-            RENKU_ANNOTATION_PREFIX + "projectName": options.get("project"),
-            "hub.jupyter.org/network-access-hub": "true"
```

Comment:
This line is necessary for the user session pod to be able to communicate with the hub after network policies were enabled by default in the hub.

Reply:
I am sorry I will put that back in. I remember seeing the PR that added that line. Not sure why I deleted it here. Good catch.